core/rest/src/main/java/org/phenotips/remote/rest/internal/DefaultRemotePatientMatchResource.java

```diff
@@ -211,4 +219,19 @@ private int getLastIndex(@Nonnull final MatchedPatientClusterView matchedCluster
         final int requestedLast = limit >= 0 ? offset + limit - 2 : lastItemIdx;
         return requestedLast <= lastItemIdx ? requestedLast : lastItemIdx;
     }
+
+    /**
+     * Saves a list of matches that were found by a remote outgoing request.
+     *
+     * @param matches list of remote similarity views
+     * @param serverId id of remote server
+     */
+    private void saveOutgoingMatches(List<RemotePatientSimilarityView> matches, String serverId)
```

Comment:
Instead of this conversion it is better to refactor `this.notificationManager.saveIncomingMatches` to accept `List<? extends PatientSimilarityView>` or something like that